### PR TITLE
Create org.dita-semia.postprocessing.json

### PR DESCRIPTION
**Pull request by PStellmann**

### CODE REVIEW

File: org.dita-semia.postprocessing.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.dita-semia.postprocessing",
+    "description": "Set pdf filename depending on content (e.g. metadata) and zipping the result.",
+    "keywords": ["zip", "filename"],
+    "homepage": "https://github.com/dita-semia/org.dita-semia.postprocessing",
+    "vers": "0.1",
```
Comment (jelovirt) [inline org.dita-semia.postprocessing.json]: This should be `0.1.0`

File: org.dita-semia.postprocessing.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": "2.4"
```
Comment (jelovirt) [inline org.dita-semia.postprocessing.json]: This should be `2.4.0` (or `>=2.4.0` if you mean works on DITA-OT 2.4.0 or newer)

File: org.dita-semia.postprocessing.json (hunk 1 above)
Comment (PStellmann) [reply org.dita-semia.postprocessing.json]: Done!

File: org.dita-semia.postprocessing.json (hunk 2 above)
Comment (PStellmann) [reply org.dita-semia.postprocessing.json]: Done!

**Pull request merged**